extensions/kubernetes-client/deployment/src/main/java/io/quarkus/kubernetes/client/deployment/DevServicesKubernetesProcessor.java

```diff
@@ -240,6 +244,8 @@ private RunningDevService startKubernetes(DockerStatusBuildItem dockerStatusBuil
 
             devServicesKube
                     .produce(new KubernetesDevServiceInfoBuildItem(container.getKubeconfig(), container.getContainerId()));
+            devServiceKubeRequest
+                    .produce(new KubernetesDevServiceRequestBuildItem(config.flavor.name()));
```

Comment:
```suggestion
```

We shouldn't be producing this here. We should be consuming it to check if other extensions have requested the dev service.